components/SnackBar/index.d.ts

```diff
@@ -1,9 +1,9 @@
-import React = require('react');
+import React from 'react';
 
-export interface SnackBarProps<T> {
+export interface SnackBarProps {
     actionTrigger?: {
         title?: string;
-        callback?: React.MouseEventHandler<T>;
+        callback?: React.MouseEventHandler<HTMLButtonElement>;
```

Comment:
Doubt: `MouseEventHandler` is the correct type to use in a callback (generic)?

Reply:
Yes, the `onClick` event belongs to `MouseEventHandler`